examples/parquet_analytics_demo.ipynb

```diff
@@ -0,0 +1,476 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "af5806d3",
+   "metadata": {},
+   "source": [
+    "# Parquet Analytics Demo Notebook"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f5d5a85",
+   "metadata": {},
+   "source": [
+    "This notebook serves as a brief demonstration of how a set of cloud-hosted parquet files can be analyzed locally using R. While the analysis in this notebook consists of contrived / toy examples, the data being operated on is in fact _real_! It is currently stored in the FHIR server as part of the PHDI Ingestion Pipeline and has been transformed and standardized by a number of preceding building blocks."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc8062d0",
+   "metadata": {},
+   "source": [
+    "To run this notebook, you'll need to configure your system to recognize the R kernel. A simple tutorial can be found here: https://irkernel.github.io/installation/. To start, make sure you've installed the latest version of R, available from the CRAN repository at https://cloud.r-project.org/index.html. Then, launch R from the terminal with the `R` command. **Important:** Do not launch R from the app--the following package installation **must** be performed from the terminal (this is because the R app does not consistently respect PATH changes set in bash profiles). Run `install.packages('IRkernel')`, then run `IRkernel::installspec()`. That's it! Your notebook software can now \"see\" the R kernel on your machine."
```

Comment:
On further reading, it doesn't appear to be. I pulled that statement from a help tutorial I used to get R working in my notebook (https://irkernel.github.io/installation/), but they don't elaborate any further. So I did some digging, and what's apparently the real issue is that the location that R installs from differs depending on whether you use CRAN (recommended), R.App / RStudio, or a package manager (namely homebrew). The frameworks and executables get put in different locations, such that if you launch from RSudio or R.App and that program isn't authorized to modify your $PATH, then changes you make to the installspec won't hold true for the root R program (the actual terminal-based interface). At least, that's what I think is happening. The recommendation then is to just install directly from CRAN (both because it's a binary installer which means no compiling from source, and because CRAN will install the libraries/frameworks/executables all in the same place which is also the first place that subsequent runs of R.App and RStudio will check when looking for the path to R) and use R within an authorized session of your terminal to make the installspec changes

TLDR: I think the advice I got in the tutorial was well-meaning but misleading. We install from CRAN and manage the installspec configuration in R terminal because it eliminates any difficulties in mucking around with $PATH configurations and will apply to all future runs of R, regardless of where they're launched from.

Reply:
I've added some more specific copy into the notebook along this point but tried to keep it simple to not overwhelm users.

Reply:
I think we should remove the parentheses around the text start with "This is because" and ending with "authorization issues". I think it makes sense to just "include" that in the paragraph. I leave that up to you though and think we can mark this as resolved either way.